utility/BUILD.bazel

```diff
@@ -12,7 +12,10 @@ load("//utility:utility_test.bzl", "utility_test")
 # Library for general-purpose utilities.
 kt_android_library(
     name = "utility",
-    srcs = glob(["src/main/java/org/oppia/android/util/**/*.kt"]),
+    srcs = glob(
+        ["src/main/java/org/oppia/android/util/**/*.kt"],
+        exclude = ["**/system/**"]
```

Comment:
I would add the full prefix here:

`        exclude = ["src/main/java/org/oppia/android/util/system/**"]`

just to be more explicit about which system directory we're excluding.

Reply:
Done